### PR TITLE
Composite Monitor

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
Update of Composite Monitor documentation

### Motivation
Audit of monitor docs

### Preview link
https://docs-staging.datadoghq.com/ruth/composite-monitors/monitors/monitor_types/composite/

### Additional Notes
Do not merge until the monitors team reviews

### CODE REVIEW

File: content/en/monitors/monitor_types/composite.md [hunk 1]
```diff
@@ -64,144 +62,122 @@ a || b && !c
...
-
-a composite monitor's query is `"1234 && 5678"`, `"!1234 || 5678"`, etc.
-
-### Configure behavior for `No Data`
-
-As with a non-composite monitor, you may configure whether or not a composite monitor triggers when it has the `No Data` status. Whatever you choose here doesn't affect the constituent monitors' `notify_no_data` settings.
+#### No data
 
-### Write a notification message
+`Do not notify` or `Notify` when the composite monitor is in a no-data state. Whatever you choose here doesn't affect the individual monitors' `Notify no data` settings.
```
Comment (unclebconnor) [inline content/en/monitors/monitor_types/composite.md]:
```suggestion
`Do not notify` or `Notify` when the composite monitor is in a no-data state. Whatever you choose here doesn't affect the individual monitors' `Notify no data` settings, but in order for a composite to alert on No Data, both the individual monitors and the composite monitor must be set to `Notify` when data is missing.
```

File: content/en/monitors/monitor_types/composite.md [hunk 2]
```diff
@@ -64,144 +62,122 @@ a || b && !c
...
+Two of the four scenarios trigger an alert, even though not all of the individual monitors have the most severe status (`Alert`).
 
-Rather than periodically sampling the current state of component monitors, composite monitors are evaluated by using a sliding window of monitor results for each component monitor (specifically, they use the most severe status from the past five minutes for each component monitor). For example, if you have a composite monitor defined as `A && B`, and the component results look like this (where the timestamps are one minute apart):
+### Component monitor status
 
-|   | T0    | T1    | T2    |
-|---|-------|-------|-------|
-| A | Alert | OK    | OK    |
-| B | OK    | Alert | Alert |
+Composite monitors are evaluated by using a sliding window of monitor results for each component monitor. Specifically, they use the most severe status from the past five minutes for each component monitor. For example, if you have a composite monitor defined as `A && B` with the following component results (timestamps are one minute apart), the composite monitor triggers at T1 even though `A` is technically in an `OK` state.
```
Comment (unclebconnor) [inline content/en/monitors/monitor_types/composite.md]:
```suggestion
Composite monitors are evaluated by using a sliding window of monitor results for each component monitor. Specifically, *they use the most severe status from the past five minutes for each component monitor*. For example, if you have a composite monitor defined as `A && B` with the following component results (timestamps are one minute apart), the composite monitor triggers at T1 even though `A` is technically in an `OK` state.
```

Comment (unclebconnor) [reply content/en/monitors/monitor_types/composite.md]: Commonly overlooked...worth highlighting

**Comment by ruthnaebeck** [comment]
> LGTM. It seems like there's not much content changing, just formatting?

Formatting, grammar, organization - just a refresh to make it more readable.

**Pull request merged**